docs/get-started/bvtz9-get-started.md

```diff
@@ -0,0 +1,406 @@
+
+This tutorial introduces you to the basics of Dagger. It walks you through installing Dagger, understanding the Dagger API and Dagger SDKs, and building a Node.js application image with Dagger.
+
+## Requirements
+
+This tutorial assumes that:
+
+- You have Docker installed and running on the host system. If not, [install Docker](https://docs.docker.com/engine/install/).
+- You have a Node.js development environment on the host system, with Node.js 14.x/16.x/18.x and `npm` installed. If not, [download and install Node.js](https://nodejs.org/en/download/).
+- You have a Docker Hub account. If not, [register for a free Docker Hub account](https://hub.docker.com/signup).
+
+## Step 1: Install Dagger
+
+The first step is to install Dagger on the host system.
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+import CodeBlock from "@theme/CodeBlock";
+import styles from "../../website/src/css/install.module.scss";
+
+<BrowserOnly>
+{() =>
+<Tabs defaultValue={
+ window.navigator.userAgent.indexOf('Linux') != -1 ? 'linux':
+ window.navigator.userAgent.indexOf('Win') != -1 ? 'windows':
+ 'macos'}
+groupId="os"
+values={[
+{label: 'macOS', value: 'macos'}, {label: 'Linux', value: 'linux'}, {label: 'Windows', value: 'windows'},
+]}>
+
+<TabItem value="macos">
+
+Instructions pending.
+
+</TabItem>
+
+<TabItem value="linux">
+
+Run the following commands:
+
+```shell
+git clone https://github.com/dagger/dagger
+cd dagger
+git checkout cloak
+go build ./cmd/cloak
+ln -sf "$(PWD)/cloak" /usr/local/bin/cloak
+```
+
+This installs `cloak` in `/usr/local/bin`. Test it with the following commands.
+
+```shell
+which cloak
+cloak is /usr/local/bin/cloak
+
+cloak version
+cloak devel (b7f3f4fec) linux/amd64
+```
+
+</TabItem>
+
+<TabItem value="windows">
+
+Instructions pending.
+
+</TabItem>
+
+</Tabs>
+}
+</BrowserOnly>
+
+## Step 2: Try the Dagger API
+
+The Dagger API is a GraphQL API for composing and running powerful pipelines with minimal effort. As a developer or operations engineer, your interaction with Dagger will be through this API, either directly by running GraphQL queries or by using a native-language Dagger SDK.
+
+### Use the query console
+
+To see this API in action, follow the steps below:
+
+1. In a new terminal, start the Dagger API server using the following command.
+
+  ```bash
+  cloak dev
+  ```
+
+  By default, this will start the Dagger API server on the local host at port 8080. Starting the server will also start an interactive GraphQL query console on the same port.
+
+1. Access the query console by browsing to `http://localhost:8080/`. Here's what you will see.
```

Comment:
Hey @vikram-dagger ! Should it be `2.` and line 106 `3.` etc.

Reply:
Not necessary, it will auto-number. See https://gist.github.com/PeeHaa/3489721